uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/BAO/AbsenceType.php

```diff
@@ -540,4 +540,39 @@ public static function getOneWithMustTakePublicHolidayAsLeaveRequest() {
 
     return null;
   }
+
+  /**
+   * Returns the date the TOIL will expire based on the date given to the method.
+   *
+   * @param int $typeId
+   *   The Absence type ID
+   * @param string $date
+   *   The date to calculate TOIL expiry date
+   *
+   * @return null|false|string
```

Comment:
Whenever you have a @return which returns different values, please always document what each type means